src/presentation/http/router/noteSettings.test.ts

```diff
@@ -368,6 +368,34 @@ describe('NoteSettings API', () => {
 
   describe('PATCH /note-settings/:notePublicId/team', () => {
     test('Update team member role by user id and note id, with status code 200', async () => {
+      test.todo('get rid of restarting database data in tests (use restart-database-data script in beforeEach)');
```

Comment:
this `test.todo` is meant for declaring testcases that are postponed. If you want to leave general todo, you can use regular comment

Reply:
done